src/main/java/org/folio/linked/data/configuration/audit/LinkedDataAuditorAware.java

```diff
@@ -0,0 +1,22 @@
+package org.folio.linked.data.configuration.audit;
+
+import java.util.Optional;
+import java.util.UUID;
+import lombok.RequiredArgsConstructor;
+import org.folio.spring.FolioExecutionContext;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.domain.AuditorAware;
+import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
+
+@Configuration
+@EnableJpaAuditing(modifyOnCreate = false)
```

Comment:
why ```false```?

Reply:
because we don't want updatedDate to be modified when a resource is updating via breaking edged and replacing the old resource

Reply:
We want updatedDate to be modified in this case right?
Consider this scenario:
Day 1: A resource got created.
Day 2: The resource got updated using PUT api. We want the update date to be modified with the current date.

Reply:
sorry for confusion, meant updatedBy of course, updated_date will be updated anyway.
if we have modifyOnCreate = true then updatedBy will be overwrited  with value from createdBy on graph update after we manually set createdBy/updatedBy, we don't want to do that in this case.